test/api_test.js

```diff
@@ -58,7 +58,7 @@ describe("Api", () => {
     describe("full URL", () => {
       var endpoints;
 
-      beforeEach(() => endpoints = api.endpoints({fullUrl: true}))
+      beforeEach(() => endpoints = api.endpoints({ fullUrl: true }))
```

Comment:
We should choose if we want spaces between curly-brackets and property name and keep it to